.github/workflows/check-coverage.ps1

```diff
@@ -0,0 +1,54 @@
+param (
+    [string]$JsonReportPath,
+    [double]$CoverageThreshold
+)
+
+$jsonContent = Get-Content $JsonReportPath -Raw | ConvertFrom-Json
+$coverageBelowThreshold = $false
+
+function GetFormattedValue($number) {
```

Comment:
nit: Get-FormattedValue to follow PS Verb-Noun convention?

Reply:
Good catch, updated! Thanks!